x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/suggestion_panel.tsx

```diff
@@ -51,46 +51,49 @@ const SuggestionPreview = ({
   }, [previewExpression]);
 
   return (
-    <EuiPanel
-      paddingSize="s"
-      data-test-subj="suggestion"
-      onClick={() => {
-        dispatch(toSwitchAction(suggestion));
-      }}
-    >
-      <EuiTitle size="xxxs">
+    <EuiToolTip content={suggestion.title}>
+      <EuiPanel
+        className="lnsSuggestionPanel__button"
+        paddingSize="none"
+        data-test-subj="lnsSuggestion"
+        onClick={() => {
+          dispatch(toSwitchAction(suggestion));
+        }}
+      >
+        {/* <EuiText size="xs">
```

Comment:
nit: commented out code

Reply:
Yeah I had left it in to remind whoever was helping with tests what the test was targeting before. I'll remove it now